src/frontends/tensorflow/src/op/placeholder.cpp

```diff
@@ -14,13 +14,25 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_placeholder_op(const NodeContext& node) {
-    auto ng_et = node.get_attribute<ov::element::Type>("dtype");
-    auto ng_shape = node.get_attribute<ov::PartialShape>("shape", ov::PartialShape());
+    auto tf_dtype = node.get_attribute<ov::element::Type>("dtype");
```

Comment:
nitpick: 
ov_dtype, ov_shape
:)

Reply:
They idea is to mark-up original attributes with a prefix `tf_`.

Reply:
they are not original attrs, they already translated to ov types

Reply:
not necessarily, because we retrieve `pad_mode` that is in string and not yet converted to ov. Here is the same thing

Reply:
but here you specify concrete ov types with <>

Reply:
Well, will be fixed in the next PR. It is nit definitely.